docs/_includes/options/selections/templating.html

```diff
@@ -27,11 +27,21 @@ <h2 id="templateSelection">
     I am using HTML in my selection template but it isn't displaying it
   </h3>
 
-  {% include options/not-written.html %}
+  If you want to use html in your selection template, you will need to return a jQuery object.
+  
+{% highlight js linenos %}
+function template(data, container) {
+  return $('<span>' + data.text + '</span>');
```

Comment:
Ideally, to avoid XSS issues, this should be something like

```js
var $template = $('<span></span>');
$template.text(data.text);
return $template;
```

That way the `data.text` is sanitized before you display it, so you don't need to worry about HTML being injected where you weren't intending it.